projects/mercury/src/metadata/common/LinkedDataValuesList.js

```diff
@@ -114,6 +114,29 @@ export const LinkedDataValuesList = (props: LinkedDataValuesListProps) => {
     const isDeleteButtonEnabled = () => property.isEditable && canEdit;
     const isAddButtonEnabled = canEdit && !maxValuesReached && AddComponent;
 
+    const renderListItem = (entry, index) => rowDecorator(entry, (
+        <Grid item xs={property.isEditable ? 10 : 12} className={classes.values}>
+            <div style={{overflow: "hidden", textOverflow: "ellipsis"}}>
+                {
+                    columnDefinition.id === LABEL_URI
+                        ? <Typography variant="h6">{columnDefinition.getValue(entry, index)}</Typography>
+                        : <Typography noWrap>{columnDefinition.getValue(entry, index)}</Typography>
+                }
+            </div>
+            {showRowDividers && <Divider />}
+        </Grid>
+    ));
+
+    const renderListLimitMessage = (entry) => rowDecorator(entry, (
+        <Grid item className={classes.values}>
+            <div>
+                <Tooltip title={"If a view tab is defined for '" + property.label + "' you can retrieve all available values there."}>
```

Comment:
I think there should be a comma before "you can ...": `If..., you can ...`

And just a suggestion, feel free to skip it: maybe for non-technical users it is not clear how to "retrieve" values... (sounds like an action to do). Maybe better to change "retrieve" to "find all available values" or "see available values""?

Reply:
ah good suggestion, write it like my grandmother would understand :D